src/Translatable.php

```diff
@@ -109,7 +111,7 @@ public function fill(array $attributes)
                 continue;
             }
 
-            $this->translation = $this->getTranslation(false);
+            $this->getTranslation(false);
```

Comment:
Well, yes. If a fallback is set to be used, this allows fill to add a new translation for the current locale. Without this line the fallback would always be used if a language didn't exist yet. (which would mean it would fail on a new object, because the fallback language wouldn't exist either).